src/client/errorMiddleware.js

```diff
@@ -1,17 +1,25 @@
 import { notify } from './app/Notification/notificationActions';
 
 function parseBlockChainError(error) {
-  let parsedError;
-  try {
-    parsedError = error
-      .split(':')[2]
-      .split('\n')[0]
-      .substring(1);
-  } catch (e) {
-    parsedError = 'Unknown error';
+  if (error.indexOf('You may only post once every 5 minutes') !== -1) {
```

Comment:
you might be able to use `_.includes(error, 'You may only post once every 5 minutes')`. Reads a bit better imo

Reply:
I like to use vanilla JS wherever possible (if it doesn't complicate code too much).